ecf/include/head.h

```diff
@@ -4,45 +4,83 @@ export PS4='+ $SECONDS + '
 
 # Variables needed for communication with ecFlow
 export ECF_NAME=%ECF_NAME%
-#export ECF_HOST=%ECF_HOST%
 export ECF_HOST=%ECF_LOGHOST%
 export ECF_PORT=%ECF_PORT%
 export ECF_PASS=%ECF_PASS%
 export ECF_TRYNO=%ECF_TRYNO%
-export ECF_RID=$LSB_JOBID
-
-# Tell ecFlow we have started
-# POST_OUT variable enables LSF post_exec to communicate with ecFlow
-if [ -d /opt/modules ]; then
-    # WCOSS TO4 (Cray XC40)
-    . /opt/modules/default/init/sh
-    module load ecflow
-    POST_OUT=/gpfs/hps/tmpfs/ecflow/ecflow_post_in.$LSB_BATCH_JID
-else
-    # WCOSS Phase 3 (Dell PowerEdge)
-    . /usrx/local/prod/lmod/lmod/init/sh
-    . /gpfs/dell1/nco/ops/nwprod/versions/ecflow_p3.ver
-    module load ips/$ips_ver
-    module load EnvVars/$EnvVars_ver
-    module load ecflow/$ecflow_ver
-    POST_OUT=/var/lsf/ecflow_post_in.$USER.$LSB_BATCH_JID
+export ECF_RID=${ECF_RID:-${PBS_JOBID:-$$}}
+export ECF_JOB=%ECF_JOB%
+export ECF_JOBOUT=%ECF_JOBOUT%
+export ecflow_ver=%ecflow_ver%
+
+if [ -d /apps/ops/prod ]; then # On WCOSS2
+####  set +x
+  echo "Running 'module reset'"
+  module reset
+  module load envvar/1.0
+  module load PrgEnv-intel/8.1.0
+  module load craype/2.7.8
+  module load intel/19.1.3.304
+  module load cray-mpich/8.1.9
+  module load cray-pals/1.0.17
+  module load python/3.8.6
```

Comment:
Curious, why `python` is required in `head.h`?

Reply:
Also.  Does every job need the mpich module?
By that extension, we could pretty much emulate `load_fv3gfs_modules.sh` in `head.h` and not have to load any module in any of the `ecf` scripts.